pkg/generator/generator.go

```diff
@@ -21,11 +22,10 @@ type InstallConfigGenerator interface {
 }
 
 type Config struct {
-	ServiceCACertPath      string `envconfig:"SERVICE_CA_CERT_PATH" default:""`
-	ReleaseImageMirror     string `envconfig:"OPENSHIFT_INSTALL_RELEASE_IMAGE_MIRROR" default:""`
-	DummyIgnition          bool   `envconfig:"DUMMY_IGNITION"`
-	InstallInvoker         string `envconfig:"INSTALL_INVOKER" default:"assisted-installer"`
-	InstallerCacheCapacity int64  `envconfig:"INSTALLER_CACHE_CAPACITY"`
+	ServiceCACertPath  string `envconfig:"SERVICE_CA_CERT_PATH" default:""`
+	ReleaseImageMirror string `envconfig:"OPENSHIFT_INSTALL_RELEASE_IMAGE_MIRROR" default:""`
+	DummyIgnition      bool   `envconfig:"DUMMY_IGNITION"`
+	InstallInvoker     string `envconfig:"INSTALL_INVOKER" default:"assisted-installer"`
```

Comment:
You removed INSTALLER_CACHE_CAPACITY from this file but it's still in use in https://github.com/openshift/assisted-service/blob/40b8a8264e6b2509af8773d2324b0b0592e0481a/openshift/template.yaml#L220

Reply:
Good catch, thank you!